# Disable upstreams catch download, we always want to use the system installed 
# version. Gentoo bug 636828.

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index 86ce5a4..9b7d4f7 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -10,21 +10,8 @@ list(APPEND CATCH_CMAKE_ARGS
     "-DNO_SELFTEST=true"
 )
 
-if(GIT_FOUND)
-    # add catch
-    ExternalProject_Add(
-        catch
-        PREFIX ${CMAKE_BINARY_DIR}/catch
-        GIT_REPOSITORY https://github.com/catchorg/Catch2.git
-        GIT_TAG v2.0.1
-        CMAKE_ARGS ${CATCH_CMAKE_ARGS}
-        LOG_DOWNLOAD 1
-        UPDATE_DISCONNECTED 1
-    )
-else()
-    # assume catch is installed in a system directory
-    add_custom_target(catch)
-endif()
+# assume catch is installed in a system directory
+add_custom_target(catch)
 
 # this interface adds compile options to how the tests are run
 # please try to keep entries ordered =)
